js/src/routes.js

```diff
@@ -16,11 +16,15 @@
 
 import {
   Accounts, Account, Addresses, Address, Application,
-  Contract, Contracts, Dapp, Dapps,
+  Contract, Contracts, Dapp, Dapps, HistoryStore,
```

Comment:
Yep that's what I think too but it's true that it feels a bit strange/out-of-place in `~/views`... @derhuerst any thoughts?